main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -1108,6 +1108,82 @@ object ResolutionError {
     override def explain(formatter: Formatter): Option[String] = None
   }
 
+  /**
```

Comment:
I notice that in all of the below errors you pass in "String" for struct and field. It would be better to parse the structsym (if it exists) and the Name.Ident.

Thanks!